CrudeOperation/src/main/java/com/example/crudeoperation/service/DeptService.java

```diff
@@ -0,0 +1,40 @@
+package com.example.crudeoperation.service;
+
+import com.example.crudeoperation.crudoperation.CrudOpearationsMethods;
+import com.example.crudeoperation.entity.DeptEntity;
+import com.example.crudeoperation.repository.DeptRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class DeptService implements CrudOpearationsMethods {
+    private final DeptRepository repo;
+
+    public DeptService(DeptRepository repo) {
+
+        this.repo = repo;
+    }
+
+    @Override
+    public DeptEntity saveDeptInfo(DeptEntity Department) {
+        return repo.save(Department);
+    }
+
+    @Override
+    public DeptEntity updateDeptInfo(DeptEntity Department) {
```

Comment:
why saveDeptInfo and updateDeptInfo are same???

Reply:
Because I am just getting whole data and save that into repo